<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="NRMI-55" name="in.novopay.cbs.apis.createorupdateproductscheme.CreateOrUpdateProductSchemeTest" tests="54" failures="0" timestamp="18 Jul 2020 06:51:04 GMT" time="47.687" errors="10">
  <testcase name="createOrUpdateProductSchemeTest" time="0.411" classname="in.novopay.cbs.apis.createorupdateproductscheme.CreateOrUpdateProductSchemeTest"/>
  <testcase name="createOrUpdateProductSchemeTest" time="0.356" classname="in.novopay.cbs.apis.createorupdateproductscheme.CreateOrUpdateProductSchemeTest"/>
  <testcase name="createOrUpdateProductSchemeTest" time="1.191" classname="in.novopay.cbs.apis.createorupdateproductscheme.CreateOrUpdateProductSchemeTest"/>
  <testcase name="createOrUpdateProductSchemeTest" time="0.370" classname="in.novopay.cbs.apis.createorupdateproductscheme.CreateOrUpdateProductSchemeTest"/>
  <testcase name="createOrUpdateProductSchemeTest" time="0.396" classname="in.novopay.cbs.apis.createorupdateproductscheme.CreateOrUpdateProductSchemeTest"/>
  <testcase name="createOrUpdateProductSchemeTest" time="0.345" classname="in.novopay.cbs.apis.createorupdateproductscheme.CreateOrUpdateProductSchemeTest"/>
  <testcase name="createOrUpdateProductSchemeTest" time="0.907" classname="in.novopay.cbs.apis.createorupdateproductscheme.CreateOrUpdateProductSchemeTest"/>
  <testcase name="createOrUpdateProductSchemeTest" time="1.079" classname="in.novopay.cbs.apis.createorupdateproductscheme.CreateOrUpdateProductSchemeTest"/>
  <testcase name="createOrUpdateProductSchemeTest" time="0.992" classname="in.novopay.cbs.apis.createorupdateproductscheme.CreateOrUpdateProductSchemeTest"/>
  <testcase name="createOrUpdateProductSchemeTest" time="0.651" classname="in.novopay.cbs.apis.createorupdateproductscheme.CreateOrUpdateProductSchemeTest"/>
  <testcase name="createOrUpdateProductSchemeTest" time="1.481" classname="in.novopay.cbs.apis.createorupdateproductscheme.CreateOrUpdateProductSchemeTest">
    <error type="java.lang.AssertionError" message="TEST FAILED!!! Assertion error..!">
      <![CDATA[java.lang.AssertionError: TEST FAILED!!! Assertion error..!
	at org.testng.Assert.fail(Assert.java:94)
	at in.novopay.cbs.utils.ApiUtils.assertAPIResponseStatus(ApiUtils.java:306)
	at in.novopay.cbs.apis.createorupdateproductscheme.CreateOrUpdateProductSchemeTest.createOrUpdateProductSchemeTest(CreateOrUpdateProductSchemeTest.java:50)
	at sun.reflect.GeneratedMethodAccessor297.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- createOrUpdateProductSchemeTest -->
  <testcase name="createOrUpdateProductSchemeTest" time="1.565" classname="in.novopay.cbs.apis.createorupdateproductscheme.CreateOrUpdateProductSchemeTest"/>
  <testcase name="createOrUpdateProductSchemeTest" time="0.967" classname="in.novopay.cbs.apis.createorupdateproductscheme.CreateOrUpdateProductSchemeTest">
    <error type="java.lang.AssertionError" message="TEST FAILED!!! Assertion error..!">
      <![CDATA[java.lang.AssertionError: TEST FAILED!!! Assertion error..!
	at org.testng.Assert.fail(Assert.java:94)
	at in.novopay.cbs.utils.ApiUtils.assertAPIResponseStatus(ApiUtils.java:306)
	at in.novopay.cbs.apis.createorupdateproductscheme.CreateOrUpdateProductSchemeTest.createOrUpdateProductSchemeTest(CreateOrUpdateProductSchemeTest.java:50)
	at sun.reflect.GeneratedMethodAccessor297.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- createOrUpdateProductSchemeTest -->
  <testcase name="createOrUpdateProductSchemeTest" time="0.497" classname="in.novopay.cbs.apis.createorupdateproductscheme.CreateOrUpdateProductSchemeTest"/>
  <testcase name="createOrUpdateProductSchemeTest" time="1.011" classname="in.novopay.cbs.apis.createorupdateproductscheme.CreateOrUpdateProductSchemeTest"/>
  <testcase name="createOrUpdateProductSchemeTest" time="1.586" classname="in.novopay.cbs.apis.createorupdateproductscheme.CreateOrUpdateProductSchemeTest"/>
  <testcase name="createOrUpdateProductSchemeTest" time="0.364" classname="in.novopay.cbs.apis.createorupdateproductscheme.CreateOrUpdateProductSchemeTest"/>
  <testcase name="createOrUpdateProductSchemeTest" time="0.300" classname="in.novopay.cbs.apis.createorupdateproductscheme.CreateOrUpdateProductSchemeTest"/>
  <testcase name="createOrUpdateProductSchemeTest" time="1.096" classname="in.novopay.cbs.apis.createorupdateproductscheme.CreateOrUpdateProductSchemeTest"/>
  <testcase name="createOrUpdateProductSchemeTest" time="0.464" classname="in.novopay.cbs.apis.createorupdateproductscheme.CreateOrUpdateProductSchemeTest"/>
  <testcase name="createOrUpdateProductSchemeTest" time="0.390" classname="in.novopay.cbs.apis.createorupdateproductscheme.CreateOrUpdateProductSchemeTest"/>
  <testcase name="createOrUpdateProductSchemeTest" time="1.916" classname="in.novopay.cbs.apis.createorupdateproductscheme.CreateOrUpdateProductSchemeTest"/>
  <testcase name="createOrUpdateProductSchemeTest" time="0.925" classname="in.novopay.cbs.apis.createorupdateproductscheme.CreateOrUpdateProductSchemeTest"/>
  <testcase name="createOrUpdateProductSchemeTest" time="0.371" classname="in.novopay.cbs.apis.createorupdateproductscheme.CreateOrUpdateProductSchemeTest">
    <error type="java.lang.AssertionError" message="TEST FAILED!!! Assertion error..!">
      <![CDATA[java.lang.AssertionError: TEST FAILED!!! Assertion error..!
	at org.testng.Assert.fail(Assert.java:94)
	at in.novopay.cbs.utils.ApiUtils.assertAPIResponseStatus(ApiUtils.java:306)
	at in.novopay.cbs.apis.createorupdateproductscheme.CreateOrUpdateProductSchemeTest.createOrUpdateProductSchemeTest(CreateOrUpdateProductSchemeTest.java:50)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- createOrUpdateProductSchemeTest -->
  <testcase name="createOrUpdateProductSchemeTest" time="0.300" classname="in.novopay.cbs.apis.createorupdateproductscheme.CreateOrUpdateProductSchemeTest"/>
  <testcase name="createOrUpdateProductSchemeTest" time="0.948" classname="in.novopay.cbs.apis.createorupdateproductscheme.CreateOrUpdateProductSchemeTest"/>
  <testcase name="createOrUpdateProductSchemeTest" time="0.969" classname="in.novopay.cbs.apis.createorupdateproductscheme.CreateOrUpdateProductSchemeTest"/>
  <testcase name="createOrUpdateProductSchemeTest" time="1.806" classname="in.novopay.cbs.apis.createorupdateproductscheme.CreateOrUpdateProductSchemeTest">
    <error type="java.lang.AssertionError" message="TEST FAILED!!! Assertion error..!">
      <![CDATA[java.lang.AssertionError: TEST FAILED!!! Assertion error..!
	at org.testng.Assert.fail(Assert.java:94)
	at in.novopay.cbs.utils.ApiUtils.assertAPIResponseStatus(ApiUtils.java:306)
	at in.novopay.cbs.apis.createorupdateproductscheme.CreateOrUpdateProductSchemeTest.createOrUpdateProductSchemeTest(CreateOrUpdateProductSchemeTest.java:50)
	at sun.reflect.GeneratedMethodAccessor297.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- createOrUpdateProductSchemeTest -->
  <testcase name="createOrUpdateProductSchemeTest" time="0.910" classname="in.novopay.cbs.apis.createorupdateproductscheme.CreateOrUpdateProductSchemeTest"/>
  <testcase name="createOrUpdateProductSchemeTest" time="0.264" classname="in.novopay.cbs.apis.createorupdateproductscheme.CreateOrUpdateProductSchemeTest"/>
  <testcase name="createOrUpdateProductSchemeTest" time="1.238" classname="in.novopay.cbs.apis.createorupdateproductscheme.CreateOrUpdateProductSchemeTest"/>
  <testcase name="createOrUpdateProductSchemeTest" time="0.462" classname="in.novopay.cbs.apis.createorupdateproductscheme.CreateOrUpdateProductSchemeTest"/>
  <testcase name="createOrUpdateProductSchemeTest" time="2.188" classname="in.novopay.cbs.apis.createorupdateproductscheme.CreateOrUpdateProductSchemeTest"/>
  <testcase name="createOrUpdateProductSchemeTest" time="0.405" classname="in.novopay.cbs.apis.createorupdateproductscheme.CreateOrUpdateProductSchemeTest"/>
  <testcase name="createOrUpdateProductSchemeTest" time="1.106" classname="in.novopay.cbs.apis.createorupdateproductscheme.CreateOrUpdateProductSchemeTest"/>
  <testcase name="createOrUpdateProductSchemeTest" time="0.909" classname="in.novopay.cbs.apis.createorupdateproductscheme.CreateOrUpdateProductSchemeTest"/>
  <testcase name="createOrUpdateProductSchemeTest" time="0.450" classname="in.novopay.cbs.apis.createorupdateproductscheme.CreateOrUpdateProductSchemeTest">
    <error type="java.lang.AssertionError" message="TEST FAILED!!! Assertion error..!">
      <![CDATA[java.lang.AssertionError: TEST FAILED!!! Assertion error..!
	at org.testng.Assert.fail(Assert.java:94)
	at in.novopay.cbs.utils.ApiUtils.assertAPIResponseStatus(ApiUtils.java:306)
	at in.novopay.cbs.apis.createorupdateproductscheme.CreateOrUpdateProductSchemeTest.createOrUpdateProductSchemeTest(CreateOrUpdateProductSchemeTest.java:50)
	at sun.reflect.GeneratedMethodAccessor297.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- createOrUpdateProductSchemeTest -->
  <testcase name="createOrUpdateProductSchemeTest" time="4.721" classname="in.novopay.cbs.apis.createorupdateproductscheme.CreateOrUpdateProductSchemeTest">
    <error type="java.lang.AssertionError" message="TEST FAILED!!! Assertion error..!">
      <![CDATA[java.lang.AssertionError: TEST FAILED!!! Assertion error..!
	at org.testng.Assert.fail(Assert.java:94)
	at in.novopay.cbs.utils.ApiUtils.assertAPIResponseStatus(ApiUtils.java:306)
	at in.novopay.cbs.apis.createorupdateproductscheme.CreateOrUpdateProductSchemeTest.createOrUpdateProductSchemeTest(CreateOrUpdateProductSchemeTest.java:50)
	at sun.reflect.GeneratedMethodAccessor297.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- createOrUpdateProductSchemeTest -->
  <testcase name="createOrUpdateProductSchemeTest" time="0.501" classname="in.novopay.cbs.apis.createorupdateproductscheme.CreateOrUpdateProductSchemeTest">
    <error type="java.lang.AssertionError" message="TEST FAILED!!! Assertion error..!">
      <![CDATA[java.lang.AssertionError: TEST FAILED!!! Assertion error..!
	at org.testng.Assert.fail(Assert.java:94)
	at in.novopay.cbs.utils.ApiUtils.assertAPIResponseStatus(ApiUtils.java:306)
	at in.novopay.cbs.apis.createorupdateproductscheme.CreateOrUpdateProductSchemeTest.createOrUpdateProductSchemeTest(CreateOrUpdateProductSchemeTest.java:50)
	at sun.reflect.GeneratedMethodAccessor297.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- createOrUpdateProductSchemeTest -->
  <testcase name="createOrUpdateProductSchemeTest" time="0.635" classname="in.novopay.cbs.apis.createorupdateproductscheme.CreateOrUpdateProductSchemeTest"/>
  <testcase name="createOrUpdateProductSchemeTest" time="0.294" classname="in.novopay.cbs.apis.createorupdateproductscheme.CreateOrUpdateProductSchemeTest"/>
  <testcase name="createOrUpdateProductSchemeTest" time="0.588" classname="in.novopay.cbs.apis.createorupdateproductscheme.CreateOrUpdateProductSchemeTest"/>
  <testcase name="createOrUpdateProductSchemeTest" time="0.401" classname="in.novopay.cbs.apis.createorupdateproductscheme.CreateOrUpdateProductSchemeTest"/>
  <testcase name="createOrUpdateProductSchemeTest" time="0.460" classname="in.novopay.cbs.apis.createorupdateproductscheme.CreateOrUpdateProductSchemeTest"/>
  <testcase name="createOrUpdateProductSchemeTest" time="0.313" classname="in.novopay.cbs.apis.createorupdateproductscheme.CreateOrUpdateProductSchemeTest"/>
  <testcase name="createOrUpdateProductSchemeTest" time="0.722" classname="in.novopay.cbs.apis.createorupdateproductscheme.CreateOrUpdateProductSchemeTest"/>
  <testcase name="createOrUpdateProductSchemeTest" time="1.023" classname="in.novopay.cbs.apis.createorupdateproductscheme.CreateOrUpdateProductSchemeTest">
    <error type="java.lang.AssertionError" message="TEST FAILED!!! Assertion error..!">
      <![CDATA[java.lang.AssertionError: TEST FAILED!!! Assertion error..!
	at org.testng.Assert.fail(Assert.java:94)
	at in.novopay.cbs.utils.ApiUtils.assertAPIResponseStatus(ApiUtils.java:306)
	at in.novopay.cbs.apis.createorupdateproductscheme.CreateOrUpdateProductSchemeTest.createOrUpdateProductSchemeTest(CreateOrUpdateProductSchemeTest.java:50)
	at sun.reflect.GeneratedMethodAccessor297.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- createOrUpdateProductSchemeTest -->
  <testcase name="createOrUpdateProductSchemeTest" time="0.783" classname="in.novopay.cbs.apis.createorupdateproductscheme.CreateOrUpdateProductSchemeTest"/>
  <testcase name="createOrUpdateProductSchemeTest" time="1.354" classname="in.novopay.cbs.apis.createorupdateproductscheme.CreateOrUpdateProductSchemeTest">
    <error type="java.lang.AssertionError" message="TEST FAILED!!! Assertion error..!">
      <![CDATA[java.lang.AssertionError: TEST FAILED!!! Assertion error..!
	at org.testng.Assert.fail(Assert.java:94)
	at in.novopay.cbs.utils.ApiUtils.assertAPIResponseStatus(ApiUtils.java:306)
	at in.novopay.cbs.apis.createorupdateproductscheme.CreateOrUpdateProductSchemeTest.createOrUpdateProductSchemeTest(CreateOrUpdateProductSchemeTest.java:50)
	at sun.reflect.GeneratedMethodAccessor297.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- createOrUpdateProductSchemeTest -->
  <testcase name="createOrUpdateProductSchemeTest" time="1.022" classname="in.novopay.cbs.apis.createorupdateproductscheme.CreateOrUpdateProductSchemeTest"/>
  <testcase name="createOrUpdateProductSchemeTest" time="1.243" classname="in.novopay.cbs.apis.createorupdateproductscheme.CreateOrUpdateProductSchemeTest"/>
  <testcase name="createOrUpdateProductSchemeTest" time="1.344" classname="in.novopay.cbs.apis.createorupdateproductscheme.CreateOrUpdateProductSchemeTest"/>
  <testcase name="createOrUpdateProductSchemeTest" time="0.324" classname="in.novopay.cbs.apis.createorupdateproductscheme.CreateOrUpdateProductSchemeTest"/>
  <testcase name="createOrUpdateProductSchemeTest" time="0.373" classname="in.novopay.cbs.apis.createorupdateproductscheme.CreateOrUpdateProductSchemeTest">
    <error type="java.lang.AssertionError" message="TEST FAILED!!! Assertion error..!">
      <![CDATA[java.lang.AssertionError: TEST FAILED!!! Assertion error..!
	at org.testng.Assert.fail(Assert.java:94)
	at in.novopay.cbs.utils.ApiUtils.assertAPIResponseStatus(ApiUtils.java:306)
	at in.novopay.cbs.apis.createorupdateproductscheme.CreateOrUpdateProductSchemeTest.createOrUpdateProductSchemeTest(CreateOrUpdateProductSchemeTest.java:50)
	at sun.reflect.GeneratedMethodAccessor297.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:343)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:305)
	at org.testng.SuiteRunner.run(SuiteRunner.java:254)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- createOrUpdateProductSchemeTest -->
</testsuite> <!-- in.novopay.cbs.apis.createorupdateproductscheme.CreateOrUpdateProductSchemeTest -->
